sound/soc/soc-core.c

```diff
@@ -1632,6 +1632,7 @@ static void soc_check_tplg_fes(struct snd_soc_card *card)
 		if (!strcmp(component->driver->ignore_machine,
 			    card->dev->driver->name))
 			goto match;
+
```

Comment:
this is unrelated. But looking at the code I'd rather remove the `goto` by using
```
	if (strcmp(component->driver->ignore_machine, card->dev->driver->name) &&
	    strcmp(component->driver->ignore_machine, dev_name(card->dev)))
		continue;
```
in a separate PR of course